azure-pipelines.yaml

```diff
@@ -17,7 +17,7 @@ schedules:
 resources:
   containers:
     - container: prime
-      image: dfdsdk/prime-pipeline:0.5.4
+      image: dfdsdk/prime-pipeline:0.5.5
```

Comment:
I guess this bump also requires a new build of Atlantis image and updating it's chart?

Reply:
It would